future-apps/payroll/.solcover.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
     norpc: true,
     copyPackages: ['@aragon/os', '@aragon/apps-finance', '@aragon/apps-vault', '@aragon/test-helpers'],
     skipFiles: [
-        'PayrollKit.sol',
```

Comment:
Should we add `examples/` to this skip list?

Reply:
sure, forgot to update this one 😅